app/assets/stylesheets/revised/pages/landing/_home.scss

```diff
@@ -12,18 +12,163 @@
 }
 
 .root-landing-banner-header {
-  background-image: image-url("landing_pages/header_bg_root-1080.jpg");
   width: 100%;
-  padding-top: 12 * $vertical-height;
-  padding-bottom: 6 * $vertical-height;
+  margin-bottom: 0;
+}
+
+.root-landing-banner-subheader {
+  width: 100%;
+  margin: 0;
+
+  .subheader-tile-link {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .subheader-tile:first-of-kind {
+    padding-right: 5px;
+  }
+
+  .subheader-tile:last-of-kind {
+    padding-left: 5px;
+  }
+
+  .subheader-tile {
+    margin: 10px auto;
+    min-width: 275px;
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .subheader-image {
+    height: 275px;
+  }
+
+  .subheader-image-caption {
+    background-color: black;
+    color: #fff;
+    font-weight: bold;
+    padding: 10px 15px;
+    text-decoration: none;
+  }
+}
+
+.center-navbar-signup-link-container {
+  display: inline-block;
```

Comment:
Something needs to happen differently when the collapse is opened - eg this should hide, or be position absolute.

![Image from iOS](https://user-images.githubusercontent.com/1235441/58662540-4c3e9b00-82df-11e9-857a-b19210becfbd.png)

Reply:
Updated in 39746d16